packages/suite-build/configs/base.webpack.config.ts

```diff
@@ -132,7 +132,7 @@ const config: webpack.Configuration = {
                     },
                 ],
             },
-            // Workers
+            // This worker loader is used for suite-desktop
```

Comment:
I am still not sure, I just commented out this part 
```
  // {
                    //     loader: 'worker-loader',
                    //     options: {
                    //         filename: 'static/worker.[contenthash].js',
                    //     },
                    // },
```
and discovery works for me using `yarn suite:dev:desktop`

Reply:
Really? I was broken for me, but let me try again.

Reply:
It definitely does not work for me if I comment that in desktop, I get error:

![image](https://github.com/user-attachments/assets/9a3513da-6fea-44aa-87b8-546bdc80843c)

Reply:
Oh, this is Graph worker. It has nothing to do with connect. So ok, I tested it with a seed with no balance so it could be it.